### DISCUSSION TITLE
No access to PPSSPP

### DISCUSSION THREAD

**Discussion by Felipefpl** [community/community#32657] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, for some unknown reason i cant access https://github.com/hrydgard/ppsspp - this started yesterday and i only the angry unicorn's page.

**Comment by sum2012** [comment, marked as answer]
https://github.com/orgs/community/discussions/32554

**Reply by Felipefpl** [reply]
Thank you for the tip.
